src/styles/components/buttons.css

```diff
@@ -9,55 +9,68 @@
     &[disabled] {
       @apply cursor-not-allowed;
     }
-   
-    &.text {
-      @apply text-blue;
-      @apply hover:text-blue-dark;
+
+    &.primary {
+      @apply bg-gradient-to-r from-purple to-purple-dark text-white;
 
       &[disabled] {
-        @apply text-gray-lighter;
+        @apply bg-gradient-to-r from-white-ish to-white-ish text-gray-lightest;
       }
+    }
+
+    &.outline-primary {
+      @apply border-2 border-primary text-primary;
 
+      &[disabled] {
+        @apply border-gray-lightest text-gray-lightest;
+      }
     }
 
-    &.primary {
-      @apply bg-blue text-white;
-      @apply hover:bg-blue-crayola;
+    &.yellow {
+      @apply bg-yellow text-primary;
 
       &[disabled] {
-        @apply bg-gray-lighter text-gray-lightest;
+        @apply bg-white-ish text-gray-lightest;
       }
     }
 
-    &.outline-primary {
-      @apply border-2 border-blue text-blue;
+    &.outline-yellow {
+      @apply border-2 border-yellow text-primary;
 
-      &:hover {
-        @apply border-blue-dark text-blue-dark;
+      &[disabled] {
+        @apply border-gray-lightest text-gray-lightest;
       }
+    }
+
+    &.aqua {
+      @apply bg-aqua text-primary;
 
       &[disabled] {
-        @apply border-gray-lighter text-gray-lighter;
+        @apply bg-white-ish text-gray-lightest;
       }
     }
 
-    &.secondary {
-      @apply bg-blue-dark text-white hover:bg-dark-charcoal;
+    &.outline-aqua {
+      @apply border-2 border-aqua text-primary;
```

Comment:
The same previous comment applies to the `secondary` class.

Reply:
Actually, adding to my comment above, the secondary color should be yellow. I'm changing it. :)